UI/src/authorizationpolicies/authorizationPolicy.js

```diff
@@ -79,5 +79,12 @@
       return (selectedAsset.isNew() && me.isOperator()) || (suggestedBoxValue && (me.isOperator() || me.isMunicipalityMaintainer()));
     };
 
+    this.isMunicipalityExcluded = function (selectedAsset) {
```

Comment:
Let's put a better name, for example, something like isStateExclusions or something like that.

Reply:
Done.